### DISCUSSION TITLE
USWDS 3.0.1

### DISCUSSION THREAD

**Discussion by thisisdano** [uswds/uswds#4674] [General]
# What's new in USWDS 3.0.1

## Improvements and bug fixes
- **Fixed errors when building from new theme files.** We updated our distributed theme files so teams building from these files can compile without errors. (https://github.com/uswds/uswds/pull/4669)

## Dependency updates
dependency | old | new
--- | --- | ---
`@storybook/addon-actions` | 6.4.18 | Removed
`@storybook/addon-postcss` | 2.0.0 | Removed
`storybook-addon-html-document` | 1.0.1 | Removed

`0` vulnerabilities in regular dependencies (dependencies for USWDS projects installed with `npm install uswds`)

Internal only: `8 moderate, 27 high` vulnerabilities in devDependencies (development dependencies)

Release ZIP SHA-256 hash: `75bcd83f886b4a3515bec16e34229521b82a5472b231d073cc32bc086e28caf1`

<hr /><em>This discussion was created from the release <a href='https://github.com/uswds/uswds/releases/tag/v3.0.1'>USWDS 3.0.1</a>.</em>

**Comment by rogeruiz** [comment]
hey @thisisdano Is there a reason why USWDS isn't being published on NPM any more? I've been trying to pull `3.0.1` from NPM and it's saying the latest version on NPM is `2.13.3`. Thanks in advance!

=> https://www.npmjs.com/package/uswds?activeTab=versions

**Reply by mejiaj** [reply]
@rogeruiz USWDS is now an org-scoped package. You can find the latest version here: https://www.npmjs.com/package/@uswds/uswds.

**Reply by rogeruiz** [reply]
Awesome thank you!
